clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -785,6 +803,11 @@ public boolean matches(AbstractRequest body) {
 
     @Test
     public void testLeaveGroupOnClose() {
+        GroupRebalanceConfig rebalanceConfig = buildRebalanceConfig(Optional.empty());
```

Comment:
This one and next one in `testMaybeLeaveGroup` need reinitialization because we need to make sure the rebalance config points to empty `group.instance.id` id so that `leaveGroupOnClose` will be false.

Reply:
Nvm, I think there are not needed.